README.md

```diff
@@ -4719,3 +4724,92 @@
       ```
       
       **[⬆ Back to Top](#table-of-contents)**
+
+278. ### What are Angular Signals?
+      A signal is a wrapper around a value that can notify interested consumers when that value changes. Signals can contain any value, from simple primitives to complex data structures.
+
+      **[⬆ Back to Top](#table-of-contents)**
+
+278. ### What are Signals?
+      A signal is a wrapper around a value that can notify interested consumers when that value changes. Signals can contain any value, from simple primitives to complex data structures.
+
+      **[⬆ Back to Top](#table-of-contents)**
+      
+279. ### Explain Angular Signals with an example. 
+      In this example, we create a signal named `count` and initialize it with a value of 0. We then connect to the signal, allowing us to be notified whenever its value changes. Finally, we add a button that increments the count when clicked.
+
+      When the button is clicked, the `incrementCount()` method is called. This method sets the new value of the `count` signal to 1. Objects connected to the signal (subscribers) are then notified of the change, and the updated value is displayed in the UI.
+
+      In typescript file
+
+      ```typescript
+      import { Component, OnInit } from '@angular/core';
+      import { signal, computed } from '@angular/core'; // Import from '@angular/core'
+
+      @Component({
+        selector: 'my-app',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+      })
+      export class AppComponent implements OnInit {
+
+        count = signal(0); // Initial value of count
+        displayedCount = computed(() => this.count()); // Computed signal for display
+
+        constructor() { }
+
+        ngOnInit() {
+          // Subscribe to displayedCount instead of count for updates in the template
+          this.displayedCount.subscribe(value => {
+            console.log('Displayed count changed to', value); // Optional logging for debugging
+          });
+        }
+
+        incrementCount() {
+          this.count.set(this.count() + 1); // Update count using set() for immutability
```

Comment:
You wouldn't use `set` here.  You would use `update`.   Delete the comment as well.